packages/nimble-components/src/table/specs/table-column-specs/table-column-duration-text-hld.md

```diff
@@ -0,0 +1,202 @@
+# Duration Table Column
+
+## Overview
+
+The duration text table column will provide a way to visualize numeric data that represents a timespan. A "timespan" is simply a numeric value representing a total number of seconds (i.e. a value of 5 means '5 seconds').
+
+### Background
+
+[Work Item](https://github.com/ni/nimble/issues/1569)
+
+### Non-goals
+
+-   APIs to configure individual aspects of the display (i.e. fractional digits for seconds display)
+-   APIs to format the duration in a variety of ways
+    -   Show duration as digital display (00:01:20.323)
+    -   Show duration with symbols indicating a quantitative value (i.e. "< 1 sec")
+
+### Risks and Challenges
+
+Formatting the duration units in the exact way that SLE presents them would require custom logic that differs from what the browser provides natively using APIs like `Intl.Numberformat.format()`. The preference here is to align with current browser behaviors for unit formatting, which includes localization.
+
+### Prior Art/Examples
+
+![ElapsedTime SLE](../spec-images/ElapsedTimeSLE.PNG)
+
+## Design
+
+Below is an example of how the `nimble-table-column-duration-text` would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-duration-text field-name="duration">Duration</nimble-table-column-duration-text>
+</nimble-table>
+```
+
+The displayed value for duration will always take the form of "\<days display>, \<hour display>, \<minute display>, \<second display>.\<fractional seconds>", where the commas are non-localized delimiters <sup>1</sup>.
+
+<sup>1.</sup> When and if we are able to adopt the `Intl.DurationFormat` to format values, we will no longer be hard-coding the commas in the string result as that object will produce the entirety of the display text, which may not match what we produce in our implementation.
+
+### API
+
+#### `TableColumnDurationText`
+
+The `TableColumnDurationText` implementation requires no additional API from what `TableColumnTextBase` provides:
+
+```ts
+// inherits field-name attribute
+public class TableColumnDurationText : TableColumnTextBase { }
+
+```
+
+This API would result in a column whose value would only show up to the 'days' unit (i.e. it would show days, hours, minutes, and seconds). We would show up to two fractional digits as needed. Formatting to change slightly from what SLE offers now using "hr" instead of "h", and "sec" instead of "s".
+
+Examples:
```

Comment:
Interestingly, most of these localized values don't align with what SLE displays for non-English today. I'm wondering if that's fine, or if we need to pull the thread a bit more to verify that we're ok with a change here. I'm not even sure who we'd check with -- localization engineers?

Reply:
Posting [on this channel](https://teams.microsoft.com/l/channel/19%3Acd7187eeafd24fc3a3d98256ba0c55c2%40thread.skype/Prj_SystemLink?groupId=0af69736-3f49-48ec-8c7e-d122f54130d5&tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e) would be a good way to ask for input from localization engineers. It'd be good to get confirmation that the strings are acceptable, but I think we should stay firm about strongly preferring the browser translations unless they're outright wrong. Let's try to avoid extra work if people suggest tweaks that are hard to support but not essential.

Reply:
I've [posted](https://teams.microsoft.com/l/message/19:cd7187eeafd24fc3a3d98256ba0c55c2@thread.skype/1696521241285?tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e&groupId=null&parentMessageId=1696521241285&teamName=Localization%20Projects&channelName=Prj_SystemLink&createdTime=1696521241285) on the channel.

Reply:
It seems the proposed direction is acceptable with the localization folks.